build/generate-svg.js

```diff
@@ -2,14 +2,36 @@
 import fastGlob from 'fast-glob';
 import {optimize} from 'svgo';
 import {parse} from 'node:path';
-import {readFile, writeFile, mkdir} from 'node:fs/promises';
+import {readFile, writeFile, mkdir, copyFile, rm} from 'node:fs/promises';
 import {fileURLToPath} from 'node:url';
+import {execSync} from 'node:child_process';
 
 const glob = (pattern) => fastGlob.sync(pattern, {
   cwd: fileURLToPath(new URL('..', import.meta.url)),
   absolute: true,
 });
 
+const removeUnwantedSvgs = () => {
```

Comment:
Should `material-icons.json` also be processed to remove these entries?

And does the `iconDefinitions` key could be removed?